src/python/grpcio/grpc/_cython/_cygrpc/records.pyx.pxi

```diff
@@ -414,6 +414,7 @@ cdef class Operation:
     self.references = []
     self._c_metadata_needs_release = False
     self._c_metadata_array_needs_destruction = False
+    self._debug_error_string = NULL
```

Comment:
This is going to conflict very significantly with [pull request 13852](https://github.com/grpc/grpc/pull/13852) that just landed. We may want to hold off on getting this merged until [issue 12531](https://github.com/grpc/grpc/issues/12531) is closer to resolution.

Reply:
Interestingly, the git merge was clean. But I assume the actual logic broke. I am fine waiting on this one. What's the ETA for resolution of #12531 ?

Reply:
Days to weeks.

Reply:
Any update on ETA?

Reply:
Is the progress towards #12531 stable enough that I can revive this PR? Or should I keep it on hold

Reply:
Stay on target... stay on target...

Reply:
amiable ping